cloudbuild.yaml

```diff
@@ -0,0 +1,28 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+timeout: 3600s
+
+options:
+  substitution_option: ALLOW_LOOSE
+
+steps:
+  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20201130-750d12f'
+    entrypoint: 'bash'
+    env:
+    - REGISTRY=$_REGISTRY
+    - GCE_PD_CSI_STAGING_IMAGE=$_REGISTRY/gcp-compute-persistent-disk-csi-driver
+    - GCE_PD_CSI_STAGING_VERSION=$_PULL_BASE_REF
+    # default cloudbuild has HOME=/builder/home and docker buildx is in /root/.docker/cli-plugins/docker-buildx
+    # set the home to /root explicitly to if using docker buildx
+    - HOME=/root
+    args:
+    - -c
+    - |
+      make build-and-push-multi-arch
+
+substitutions:
+  _REGISTRY: 'fake.repository/registry-name'
```

Comment:
How does this registry work?

In particular I think in the filestore driver we seem to just get it from the project (q.v.)

Reply:
this is set in the prow job, to configure the registry name like `gcr.io/my-registry` we did that in the k/release to support other registries in the cloudbuild for any other users that are using that. But we can make it static in this case. I don't have a strong opinion on that.

Reply:
reverted and make similar in what we have for gcp-filestore-csi-driver